ansible/local_testing/README.md

```diff
@@ -1,28 +1,52 @@
 # Testing Locally
 
+This section contains the necessary steps in order to be able to setup virtual machines that mimic our netcup servers and be able to run the ansible roles/playbooks we have on them.
+
 ### Requirements
 
 - [Vagrant](https://developer.hashicorp.com/vagrant/docs/installation)
 - [VirtualBox](https://www.virtualbox.org/wiki/Downloads)
 
-## Get Started
 
-```shell
-vagrant up                  # This will take a while
-vagrant ssh                 # Get a shell, password=vagrant
-
-# inside control VM
-/vagrant/scripts/push-keys  # Push the control VM's ssh key to all nodes
-
-# run ansible
-cd infra
-# if you're on Windows, the `infra` directory will be mounted with permissions set to 777
-# which ansible will reject as insecure
-# export the below environment variable to force anisble to accept the writable config
-# export ANSIBLE_CONFIG='/home/vagrant/infra/ansible.cfg'
+### Getting started
+
+1. Creating the virtual boxes
+
+On your host machine, run the following:
+
+```bash
+cd ansible/local_testing
+vagrant up
+```
+
+This will create all the virtual boxes on which we will run our ansible playbooks.
+
+2. Push the control VM's ssh key to all nodes
+
+
+```bash
+vagrant ssh  # This will give you terminal access to the `control` VM, which we will use as the ansible control node.
+
+cd /home/vagrant/infra
+bash ./local_testing/scripts/push-keys
+```
+
+> If you're on windows, you might run into line ending issues when running the `push-keys` script.
+> To fix it, run this **while still inside the ssh session*: `sed -i 's/\r$//' /home/vagrant/infra/local_testing/scripts/push-keys`
```

Comment:
I'm not on Windows, but shouldn't the file be checked in without `\r` line endings and thus not cause any issues?
Although it might still, because it's Windows. I'm not going to act like it makes sane decisions

Reply:
That's the theory indeed, but when I tried  to push the SSH keys it started complaining :/

Should we maybe add it to .gitannotations ?

Reply:
Hmm, I think that's worth a shot! If we can make the local setup easier
in any way we should.